doc/markus-contributors.txt

```diff
@@ -91,7 +92,7 @@ Horatiu Halmaghi
 Huiyu Sun
 Ian Smith
 Ibrahim Shahin
-Ido Ben Haim
+Igit revert HEADdo Ben Haim
```

Comment:
This line contains an error. Please make sure to check your work carefully when making commits, using `git diff` (and the GitHub pull request interface).